### ISSUE TITLE
reverse logic of test-sslv2-connection.py

### ISSUE THREAD

**Issue by tomato42** [tomato42/tlsfuzzer#658]
`test-sslv2-connection.py` logs failures (because of "failure to connect") but returns 0 exit code (because no sslv2 is expected)

inverse this logic, so that failures to connect are expected and it lists `fail: 0` at end of execution

**Comment by Stonemason5040** [comment]
should the fail to connect count as xfailed or  passed

**Comment by tomato42** [comment]
no, I think we should make the test behave more standard like: make aborted connections report pass and successful report failure
